dashboard/Dockerfile

```diff
@@ -19,10 +19,9 @@ RUN apt-get update && \
     && apt-get -y clean \
     && rm -rf /var/lib/apt/lists/*
 COPY package.json /dashboard/
-RUN cd /dashboard && npm install
-COPY bower.json /dashboard/
-RUN cd /dashboard && ./node_modules/.bin/bower install --allow-root
+RUN cd /dashboard && npm i yarn && node_modules/yarn/bin/yarn install --ignore-scripts
```

Comment:
wy do we need to run yarn a first time ?

Reply:
I suppose to install dependencies, and the second run is to run build

Reply:
Sorry. I'm just having a look at it.
Yes. @akurinnoy is a quite right